packages/respect-core/src/run.ts

```diff
@@ -57,13 +59,15 @@ export async function run(options: RespectOptions): Promise<RunFileResult[]> {
 async function runFile({
   options,
   startedAt,
+  executedStepsCount,
   collectSpecData,
 }: {
   options: RunOptions;
   startedAt: number;
+  executedStepsCount: ExecutedStepsCount;
   collectSpecData?: CollectFn;
 }): Promise<RunFileResult> {
-  const result = await runTestFile(options, collectSpecData);
+  const result = await runTestFile({ options, collectSpecData, executedStepsCount });
```

Comment:
I think we can declare [this line](https://github.com/Redocly/redocly-cli/pull/2254/files#diff-506a59c0d66ea6dbc12dac72910481ff7f53671c50a900e998285c236816a088R35) here.

Reply:
Or [here](https://github.com/Redocly/redocly-cli/pull/2254/files#diff-cdff60738b3360481be05ac2a341b18d47ced550d3beea9ef1cf1ecfcefcf89bR33).

Reply:
Or even [here](https://github.com/Redocly/redocly-cli/pull/2254/files#diff-cdff60738b3360481be05ac2a341b18d47ced550d3beea9ef1cf1ecfcefcf89bR63).

Reply:
And therefore pass the `executedStepsCount` down fewer floors.